pylana/logs.py

```diff
@@ -137,6 +137,32 @@ def upload_event_log_df(self, name: str,
                                      case_attributes=df_cases.to_csv(index=False),
                                      case_attribute_semantics=case_semantics, **kwargs)
 
+    def upload_event_log_file(self, name: str,
+                            event_file_path: str,
+                            case_file_path: str,
+                            event_semantics: str,
+                            case_semantics: str) -> Response:
+        """
+        upload an event log from file location with semantics
+
+        Semantic file via path
```

Comment:
We should elaborate a bit more what we expect the files to be, if we do

```python
"""
upload an event log and case semantics read from file locations

files require to be utf-8 encoded
"""
```

Reply:
added some more, but they don't need to be utf-8 encoded